src/components/GTFOBins/GtfoMain.jsx

```diff
@@ -0,0 +1,127 @@
+import { Wrapper } from "../Dashboard/Profile/ProfileElements";
+import React, { useState, useEffect } from "react";
+import { Data } from "./Data";
+// import { Search } from "../Labs/MainElement";
+import { FunctionData } from "./_gtfoJson";
+
+export default function GtfoBin() {
+    const [activeFunction, setActiveFunction] = useState("All");
+
+    useEffect(() => {}, [activeFunction]);
+
+    function handleActiveFunction(data) {
+        setActiveFunction(data);
+    }
+
+    return (
+        <Wrapper style={{ width: "60%" }}>
+            <label style={{ fontSize: "48px", fontWeight: "bold", letterSpacing: "5px" }}>BinExploit</label>
+
+            <div style={{ display: "block", width: "100%", textAlign: "center", margin: "30px" }}>
+                <button
+                    style={{ padding: "10px", border: "solid 1px", margin: "10px" }}
+                    onClick={() => {
+                        handleActiveFunction("All");
+                    }}
+                >
+                    All
+                </button>
+
+                {Object.keys(FunctionData).map((data) => (
+                    <button
+                        style={{ padding: "10px", border: "solid 1px", margin: "10px" }}
+                        key={data}
+                        onClick={() => {
+                            handleActiveFunction(data);
+                        }}
+                    >
+                        {data}
+                    </button>
+                ))}
+            </div>
+            <div
+                style={{
+                    display: "flex",
+                    flexDirection: "column",
+                    justifyContent: "center",
+                    alignContent: "center",
+                    padding: "30px",
+                }}
+            >
+                {/* <Search
+                    id="Search"
+                    placeholder={"Search among " + Object.keys(Data[0]).length + " Binaries"}
+                    style={{ marginBottom: "20px", marginLeft: "0" }}
+                /> */}
+                {Data.map((data, index) =>
+                    Object.keys(data).map((keys) => {
```

Comment:
better to change to `key` instead of `keys` ,because you get all the keys and iterate over every key.